readme.md

```diff
@@ -352,6 +352,38 @@ Create a new `ky` instance with some defaults overridden with your own.
 
 In contrast to `ky.create()`, `ky.extend()` inherits defaults from its parent.
 
+You can pass headers as a `Headers` instance or a plain object.
+
+You can remove a header with `.extend()` by passing the header with an `undefined` value.
+Passing `undefined` as a string removes the header only if it comes from a `Headers` instance.
+
+```js
+import ky from 'ky';
+
+const url = 'https://sindresorhus.com';
+
+const original = ky.create({
+	headers: {
+		rainbow: 'rainbow',
+		unicorn: 'unicorn'
+	}
+});
+
+const extended = original.extend({
+	headers: {
+		rainbow: undefined
+	}
+});
+
+const response = await extended(`${url}/test`).json();
```

Comment:
Missing context. What is `${url}/test`? Use `https://httpbin.org/anything` instead. Same for the docs in TS definitions file.

Reply:
Will do.